libraries/base/Control/Monad.hs

```diff
@@ -231,7 +231,7 @@ inline the entire definition (as happens for foldr, for example) thereby
 specialising for the particular action.
 
 For further information, see this Trac comment, which includes side-by-side
-Core: https://ghc.haskell.org/trac/ghc/ticket/11795#comment:6
+Core: https://gitlab.haskell.org/ghc/ghc/-/issues/11795#comment:6
```

Comment:
As noted earlier, the `#` suffixes are not updated. Updating the bits would require a lot of effort and I'd rather leave the human readable text than risk getting the update wrong.